wrappers/javascript/anoncreds-nodejs/src/NodeJSAnoncreds.ts

```diff
@@ -723,6 +726,8 @@ export class NodeJSAnoncreds implements Anoncreds {
     const returnValue = handleReturnPointer<{ data: Buffer; len: number }>(ret)
     const output = new Uint8Array(byteBufferToBuffer(returnValue))
 
+    this.nativeAnoncreds.anoncreds_buffer_free(returnValue.data)
```

Comment:
Is a buffer also copied and not a ref?

Reply:
It would depend on how `ffi-napi` `ref-napi` handles this. It seems to me that they copy the data, but I haven't checked the code, yet.